src/GeositeFramework/js/Map.js

```diff
@@ -33,14 +36,10 @@ require(['use!Geosite',
     'use strict';
 
     function getSelectedBasemapLayer(model, esriMap) {
-        // Return an ESRI layer object for the currently-selected basemap spec
+        // Add the selected basemap to the Esri map
         var basemap = getSelectedBasemap(model);
-        if (basemap.layer === undefined) {
-            // This basemap has no layer yet, so make one and cache it
-            basemap.layer = new TileLayer(basemap.url);
-            esriMap.addLayer(basemap.layer);
-        }
-        return basemap.layer;
+        basemap.layer = new TileLayer(basemap.url);
```

Comment:
Investigated this a little bit more and it looks like we might want to adjust how we're creating the basemap layers in accordance with the new API. I think right now this PR takes advantage of just setting each layer individually and directly, but the better way to handle this would be to create an initial `Basemap` object using the `baseLayers` specified in `region.json` -- https://developers.arcgis.com/javascript/latest/api-reference/esri-Basemap.html#baseLayers -- and then adjust everything to toggle those layers on changing the dropdown as it was working before. 

I think what may be happening on this branch is that they're being added as "operational layers" rather than basemap layers, and the new API seems to want to split those into two categories: 

https://developers.arcgis.com/javascript/latest/guide/migrating/index.html#mapandlayer

For now I'm going to restore the code I cut, but comment it out with a note about updating how it's handled and a link to the docs. I'll make a new issue for setting them up differently, as I think that would also be able to solve setting the basemaps from `region.json` as was the case before.